pom.xml

```diff
@@ -443,9 +443,16 @@
       </exclusions>
     </dependency>
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <version>${junit.version}</version>
+      <groupId>org.powermock</groupId>
```

Comment:
This same dependencies are used for Wrangler https://github.com/data-integrations/wrangler/blob/a26fe155e4895b262e95852c6051f178764c134c/wrangler-core/pom.xml#L86

Reply:
I have added the jUnit dependency again.